common-lib/src/cmr/common/api/web_server.clj

```diff
@@ -46,44 +47,92 @@
   report in Cubby can be in the 5 to 10 MB range."
  (* 50 ONE_MB))
 
-(defn- routes-fn-verify-size
+(def buffer-size
```

Comment:
I am assuming that you get to this number after considering our average request size and playing with different settings with benchmark testing.